client/Lykke.Service.FeeCalculator.Client/AutorestClient/IFeeCalculatorAPI.cs

```diff
@@ -60,8 +60,34 @@ public partial interface IFeeCalculatorAPI : System.IDisposable
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        Task<HttpOperationResponse<object>> AddFeeWithHttpMessagesAsync(FeeModel model = default(FeeModel), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<HttpOperationResponse<ErrorResponse>> AddCashoutFeeWithHttpMessagesAsync(CashoutFeeModel model = default(CashoutFeeModel), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Why all methods return ErrorResponse?

Reply:
because of
[ProducesResponseType(typeof(void), (int) HttpStatusCode.OK)]
[ProducesResponseType(typeof(ErrorResponse), (int) HttpStatusCode.BadRequest)]